fastlane/lib/fastlane/actions/apple_pay_cert.rb

```diff
@@ -0,0 +1,153 @@
+require 'spaceship'
+
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class ApplePayCertAction < Action
+      def self.run(params)
+        UI.message("Starting login with user '#{params[:username]}'")
+        Spaceship.login(params[:username], nil)
+        Spaceship.client.select_team(team_id: params[:team_id], team_name: params[:team_name])
+        UI.message("Successfully logged in")
+
+        create_certificate(params)
+      end
+
+      def self.create_certificate(params)
+        UI.important("Creating a new Apple Pay certificate.")
+        csr, pkey = Spaceship.certificate.create_apple_pay_certificate_signing_request
+
+        begin
+          cert = certificate.create!(csr: csr, bundle_id: params[:merchant_bundle_id])
+        rescue => ex
+          if ex.to_s.include?("You already have a current")
+            UI.message(ex.to_s)
+            UI.user_error!("You already have 2 apple pay certificates for this merchant. You'll need to revoke an old certificate to make room for a new one")
+          else
+            raise ex
+          end
+        end
+
+        x509_certificate = cert.download
+        filename_base = params[:merchant_bundle_id] # PEM.config[:pem_name] || "#{PEM.config[:app_identifier]}"
```

Comment:
What does this comment mean here?

Reply:
https://github.com/fastlane/fastlane/pull/12466/files#diff-a0e4b204f34e72aedc94d3610c0af05dR20

Would I be able to provide the CSR from my Stripe account to this? Apple Pay isn't going to work in my case unless I use Stripe's CSR. I don't believe I have the option to upload the CSR to Stripe - I have to use theirs I think.

Reply:
removed the comment. It was irrelevant.